### ISSUE TITLE
Thanks and asked how to train

### ISSUE THREAD

**Issue by revalue-it** [ParaN3xus/my-unimernet#1]
Thank you for your work, this is more concise than the official one, please how to do the training, can you give the code for the training?

**Comment by ParaN3xus** [comment]
Thank you for your appreciation. Since I've packaged this model to be compatible with `VisionEncoderDecoder`, you can refer to the official transformers training code ([with native PyTorch](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Fine_tune_TrOCR_on_IAM_Handwriting_Database_using_native_PyTorch.ipynb), [with `Seq2SeqTrainer`](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Fine_tune_TrOCR_on_IAM_Handwriting_Database_using_Seq2SeqTrainer.ipynb)) for training.

It's important to note that `UniMERNetTrainImageProcessor` includes the image augmentation used in the official implementation, so it's recommended to use it during training, while using the `UniMERNetEvalImageProcessor` (without image augmentation) during evaluation.

The official training code also uses linear learning rate warmup and cosine as the lr scheduler. If you want to reproduce their results, these are essential components.

I have a personal training program at https://github.com/ParaN3xus/typress/commits/typress-unimernet/, but it's not general-purpose and not well documented. However, you can use it as a reference.
